samples/drivers/spi_flash/boards/stm32h573i_dk.overlay

```diff
@@ -5,9 +5,9 @@
  */
 
 &xspi1 {
-       /* request 57 for XSPI1 */
+	/* request 57 for XSPI1 */
 	dmas = <&gpdma1 4 57 STM32_DMA_PERIPH_TX
-		&gpdma1 5 57 STM32_DMA_PERIPH_RX>;
+			&gpdma1 5 57 STM32_DMA_PERIPH_RX>;
```

Comment:
Is tabs and then a few spaces allowed to align entries? If so, this and other similar changes should probably remain unchanged.

Reply:
I was performing the alignment based on 4 character tabs, which I now see is not the default for dts source. I will do another parse over the files